test/test_other.py

```diff
@@ -9190,21 +9191,24 @@ def test_ctor_ordering(self, args):
   # test debug info and debuggability of JS output
   @crossplatform
   def test_binaryen_debug(self):
-    for args, expect_emit_text, expect_clean_js, expect_whitespace_js, expect_closured in [
-        (['-O0'], False, False, True, False),
-        (['-O0', '-g1'], False, False, True, False),
-        (['-O0', '-g2'], False, False, True, False), # in -g2+, we emit -g to asm2wasm so function names are saved
-        (['-O0', '-g'], True, False, True, False),
-        (['-O0', '--profiling-funcs'], False, False, True, False),
-        (['-O1'],        False, False, True, False),
-        (['-O2'],        False, True,  False, False),
-        (['-O2', '-gz'], False, True,  False, False), # -gz means debug compression, it should not enable debugging
-        (['-O2', '-g1'], False, False, True, False),
-        (['-O2', '-g'],  True,  False, True, False),
-        (['-O2', '--closure=1'],         False, True, False, True),
-        (['-O2', '--closure=1', '-g1'],  False, True, True,  True),
+    for args, expect_clean_js, expect_whitespace_js, expect_closured in [
```

Comment:
lines 9200, 9202, and 9203 are new, the rest are the removal of the unused 'expect_clean_js' parameter.

Reply:
Was "expect_clean_js" a typo of "expect_emit_text"? (the latter seems to be the one removed)